### DISCUSSION TITLE
Caso d'uso per cambio cognome/nome da Tribunale.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#913] [Miglioramenti] [answered]
Buongiorno,

portiamo alla vostra attenzione il caso relativo alla richiesta di trascrizione di una sentenza in merito al cognome del minore riconosciuto, proveniente da un Tribunale.
Il minore viene riconosciuto dal genitore, dopo che l'altro genitore l’aveva già riconosciuto alla nascita. I genitori esprimono la loro preferenza circa il nuovo cognome del bambino e tale preferenza viene trasmessa insieme agli atti al Tribunale che decide ed emette una sentenza che può modificare o mantenere il cognome.
Tale sentenza viene ora trascritta nei registri di nascita in parte II serie B.
Chiediamo indicazioni relativamente al caso d’uso da utilizzare per questa tipologia di atto.
Dall’analisi che abbiamo fatto riteniamo che il caso d’uso Trascr_012 contenga i metadati corretti ma abbiamo incontrato difficoltà ad utilizzare il caso sopra descritto a causa dell’obbligatorietà del documento “Decreto Prefettizio” e per la descrizione puntuale attualmente prevista.

In attesa di riscontro,

cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Pensavamo di generalizzare il caso d'uso, modificandone le decrizioni di : 
* Caso d'uso in "decreto prefettizio o senteza..."
* Allegato "Decreto prefettizio / Senteza"
In modo che possa essere usato in modo chiaro in entrambe le casistiche.

Saluti

Assistenza ANSC

**Comment by gcuci** [comment]
ok, con questa impostazione, possiamo gestire con il Trascr_012, anche il caso di Sentenza da Tribunale.

Attendiamo la formalizzazione della modifica.

**Comment by asqueglia** [comment]
Buonasera con la versione 1.32 è stata corretta la segnalazione. Chiudiamo pertanto la issue
